R/as_factor.R

```diff
@@ -32,3 +35,18 @@ as_factor.character <- function(x, ...) {
     label = attr(x, "label", exact = TRUE)
   )
 }
+
+#' @rdname as_factor
+#' @export
+as_factor.numeric <- function(x, ...) {
+
+  x <- as.character(x)
```

Comment:
I think this would be simpler as `factor(x, levels = sort(unique(x))`?

Reply:
You're right; the character step was left over from when I wanted labels in order of appearance.

Now that you mention it - `factor(x)` is good enough, levels are in numeric order by default.